policyengine_us/parameters/gov/states/household/state_ctcs.yaml

```
@@ -0,0 +1,28 @@
+description: All state Child Tax Credits.
+values:
+  0000-01-01:
+    - az_dependent_tax_credit  # Arizona Dependent Tax Credit
+    - ca_yctc  # California Young Child Tax Credit
+    - co_ctc  # Colorado Child Tax Credit
+    - co_family_affordability_credit  # Colorado Family Affordability Credit
+    - ct_child_tax_rebate  # Connecticut Child Tax Rebate
+    - dc_ctc  # District of Columbia Child Tax Credit
+    - id_ctc  # Idaho Child Tax Credit
```

```suggestion
    - id_ctc  # Idaho Child Tax Credit
```
The credit will be discontinued in 2026